public/wish.webmanifest.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "wish",
+    "description": "Create your dream wish list with our easy-to-use platform. Whether you're planning a wedding, birthday, or simply treating yourself, our wish list feature allows you to curate a personalized collection of items you desire. ",
+    "start_url": "https://wish-local.xnor.one",
+    "theme_color": "white",
+    "background_color": "white",
+    "display": "standalone",
+    "dir": "auto",
+    "orientation": "portrait",
+    "scope": "https://wish-local.xnor.one",
+    "lang": "fa",
+    "purpose": "",
```

Comment:
could you please remove this, if its gonna be empty, so it should not be presented in the json